index.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap" rel="stylesheet">
+<link rel="stylesheet" href="style.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" 
+integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" 
+crossorigin="anonymous" referrerpolicy="no-referrer">
+<title>73rd Fifa congress</title>
+</head>
+<body>
+  <header class="header-header">
+    <ul class="header-list">
+      <li>
+        <a href="#">
+          <i class="fa-brands fa-facebook"></i>
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          <i class="fa-brands fa-twitter"></i>
+        </a>
+      </li>
+      <li class="header-link"><a href="#Mypage">My Page</a></li>
+      <li class="header-link"><a href="#">English</a></li>
+    </ul> 
+  </header>
+  <nav class="nav">
+    <i class="fa-solid fa-bars menu-menu" id="menu-icon"></i>
+    <img src="./images/page-images/FIFA.png" alt="73rd Congress logo" class="img-logo">
```

Comment:
- [ ] Kindly ensure that the project's logo is a link that links to the home page.

![image](https://user-images.githubusercontent.com/86318284/229805841-577f2e95-d8fb-4a9c-a082-fc16405a6522.png)

**Kindly implement this in the about page as well.**